core/src/main/java/org/elasticsearch/action/admin/indices/create/CreateIndexRequest.java

```diff
@@ -440,6 +443,28 @@ public CreateIndexRequest updateAllTypes(boolean updateAllTypes) {
         return this;
     }
 
+    public int waitOnActiveShards() {
```

Comment:
There is a superclass ReplicationRequest where we should put these methods (as they apply to all the replication requests).